homeassistant/components/group/__init__.py

```diff
@@ -407,21 +401,22 @@ async def async_added_to_hass(self) -> None:
         """Register listeners."""
 
         async def _update_at_start(_):
-            await self.async_update()
+            self.async_update()
             self.async_write_ha_state()
 
-        self.hass.bus.async_listen_once(EVENT_HOMEASSISTANT_START, _update_at_start)
+        start.async_at_start(self.hass, _update_at_start)
 
-    async def async_defer_or_update_ha_state(self) -> None:
+    @callback
+    def async_defer_or_update_ha_state(self) -> None:
         """Only update once at start."""
-        if self.hass.state != CoreState.running:
+        if not self.hass.is_running:
             return
 
-        await self.async_update()
+        self.async_update()
         self.async_write_ha_state()
 
     @abstractmethod
-    async def async_update(self) -> None:
```

Comment:
Please pick a different method name. `async_update`, if defined, is expected to be async. This will cause an exception if a user would use the `homeassistant.update_entity` service.